openmetadata-ui/src/main/resources/ui/src/components/common/RichTextEditor/CustomHtmlRederer/CustomHtmlRederer.tsx

```diff
@@ -23,6 +23,12 @@ import {
   LinkMdNode,
   MdNode,
 } from '@toast-ui/editor';
+import CodeMirror from 'codemirror';
+import 'codemirror/addon/runmode/runmode';
+import 'codemirror/mode/javascript/javascript';
+import 'codemirror/mode/python/python';
+import 'codemirror/mode/sql/sql';
+import 'codemirror/mode/yaml/yaml';
```

Comment:
🤔 Consideration
- I have added some modes (CodeMirror's language support) here.
  - `sql` and `javascript`: already imported [here](https://github.com/open-metadata/OpenMetadata/blob/d678e04e2ef81f847d3d1b8e60586addad0c0403/openmetadata-ui/src/main/resources/ui/src/components/Database/SchemaEditor/SchemaEditor.tsx#L24-L25), but explicitly added.
  - `python` and `yaml`: added them because we actually needed them in our use case.
- In general, adding support for other languages would be a trade-off with bundle size, so it would need to be discussed (but I don't think it is particularly large).

Reply:
Hello @fuzmish, thanks for the contribution! This is a great start, and you’ve covered many common languages. Could you please add Java to this PR as well? It’s widely used, and some users might need support for it.

Reply:
@Sachin-chaurasiya Thank you for your feedback. In 71baf30 I have added lanaguage support for Java (it seems to be supported in 'clike' (C-like language) mode in codemirror).